tests/components/google_assistant/test_trait.py

```diff
@@ -3214,6 +3214,48 @@ async def test_channel(hass: HomeAssistant) -> None:
     assert len(media_player_calls) == 1
 
 
+async def test_air_quality_description_for_aqi(hass: HomeAssistant) -> None:
+    """Test air quality description for a given AQI value."""
+    trt = trait.SensorStateTrait(
+        hass,
+        State(
+            "sensor.test",
+            100.0,
+            {
+                "device_class": sensor.SensorDeviceClass.AQI,
+            },
+        ),
+        BASIC_CONFIG,
+    )
+
+    assert trt._air_quality_description_for_aqi(0) == "healthy"
```

Comment:
State is always a string type. So it will never contain a numeric value. So you are sort testing the wrong thing here. Should be "0", "75", ..

Reply:
ah I had missed that - makes sense. fixing!